client/src/containers/CampaignAdmin/CampaignAdmin.scss

```diff
@@ -173,3 +173,11 @@
 .campaign-toolbar {
   flex-direction: row-reverse;
 }
+
+.campaign--toolbar__extra {
+  width: auto;
+
+  button:last-of-type {
+    margin-right: -10px;
+  }
+}
```

Comment:
Same CSS is used in asset admin on a class called `asset-admin__toolbar-extra` and is needed to pull the search button to the right of the toolbar